test/unit/uk/gov/hmrc/apidefinition/models/APIDefinitionSpec.scala

```diff
@@ -252,6 +253,103 @@ class APIDefinitionSpec extends UnitSpec {
       intercept[RuntimeException] { Json.parse(body).as[APIDefinition] }
     }
 
+    "read from JSON when the API categories are defined but empty" in {
```

Comment:
What about pulling the JSON out into a function that accepts category values for inclusion so that the JSON isn't duplicated?